frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -60,24 +60,26 @@ export const Onboarding = () => {
 
   const exampleDashboardId = useSetting("example-dashboard-id");
 
-  const iframeRefs = useMemo(() => {
+  const itemRefs = useMemo(() => {
     return {
-      "x-ray": createRef<HTMLIFrameElement>(),
-      notebook: createRef<HTMLIFrameElement>(),
-      sql: createRef<HTMLIFrameElement>(),
-      dashboard: createRef<HTMLIFrameElement>(),
-      subscription: createRef<HTMLIFrameElement>(),
-      alert: createRef<HTMLIFrameElement>(),
+      database: createRef<HTMLDivElement>(),
+      invite: createRef<HTMLDivElement>(),
+      "x-ray": createRef<HTMLDivElement>(),
+      notebook: createRef<HTMLDivElement>(),
+      sql: createRef<HTMLDivElement>(),
+      dashboard: createRef<HTMLDivElement>(),
+      subscription: createRef<HTMLDivElement>(),
+      alert: createRef<HTMLDivElement>(),
     };
   }, []);
 
-  type IframeKeys = keyof typeof iframeRefs;
+  type ItemKeys = keyof typeof itemRefs;
```

Comment:
The type contains a single key (with multiple possible values), so should have a singular name:

```suggestion
  type ItemKey = keyof typeof itemRefs;
```